sdk/monitor/monitor-opentelemetry/src/metrics/quickpulse/liveMetrics.ts

```diff
@@ -184,13 +184,17 @@ export class LiveMetrics {
       endpointUrl: parsedConnectionString.liveendpoint || DEFAULT_LIVEMETRICS_ENDPOINT,
       instrumentationKey: parsedConnectionString.instrumentationkey || "",
       credential: this.config.azureMonitorExporterOptions.credential,
-      credentialScopes: this.config.azureMonitorExporterOptions.credentialScopes,
+      credentialScopes:
+        this.config.azureMonitorExporterOptions.credentialScopes ||
+        parsedConnectionString.aadaudience,
     });
     const exporterOptions: QuickpulseExporterOptions = {
       endpointUrl: parsedConnectionString.liveendpoint || DEFAULT_LIVEMETRICS_ENDPOINT,
       instrumentationKey: parsedConnectionString.instrumentationkey || "",
       credential: this.config.azureMonitorExporterOptions.credential,
-      credentialScopes: this.config.azureMonitorExporterOptions.credentialScopes,
+      credentialScopes:
+        this.config.azureMonitorExporterOptions.credentialScopes ||
```

Comment:
Is there ever a situation when both this.config.azureMonitorExporterOptions.credentialScopes & parsedConnectString.aadAudience is defined?

Reply:
Yes, and connection string should take priority, @JacksonWeber please update

Reply:
Fixed